core/node/da_clients/src/eigen/sdk.rs

```diff
@@ -145,15 +154,14 @@ impl RawEigenClient {
             .unwrap()
             .payload
             .ok_or_else(|| anyhow::anyhow!("No payload in response"))?;
+        drop(client_lock);
```

Comment:
This is not really nice, we could just do it all in a single line like we used to in our older implementation, and just call the disperse data before the disperse blob authenticated, since it doesnt depend on it

Reply:
The use of `drop` is widely used across the proyect's code (connection to the database for example).